docs/en/deploy/Running-Alluxio-On-Docker.md

```diff
@@ -287,6 +287,12 @@ Congratulations, you've deployed a basic Dockerized Alluxio cluster! Read on to
 
 ## Advanced Setup
 
+### Launch Alluxio with Java11
+Starting from v2.9.0, Alluxio processes can be launched with Java 11 inside Docker containers. The new docker image, `alluxio/alluxio-jdk11` is available
+in Dockerhub. Java 11 has more GC options and potential performance gain.
```

Comment:
Do we want to mention the potential performance gain?

Reply:
Let's remove this part till we have benchmark result in public.

Reply:
sounds good, i will update this in a batch of small docfixes